semcore/data-table/src/Head.tsx

```diff
@@ -48,10 +52,86 @@ class Head extends Component<AsProps> {
     }
   };
 
-  refColumn = (props: Column['props']) => (ref: HTMLElement) => {
-    setRef(props.ref, ref);
-    if (props.forwardRef) {
-      setRef(props.forwardRef, ref);
+  refSort = (active: boolean) => (ref: HTMLElement | null) => {
+    if (ref) {
+      this.sortWrapperRefs.set(ref, active);
+    }
+  };
+
+  refColumn = (column: Column) => (ref: HTMLElement | null) => {
```

Comment:
Let's rename `refSort` and old `refColumn` to more correct `makeSortRefHandler` and `makeColumnRefHandler`

Reply:
renamed